authenticate/handlers.go

```diff
@@ -603,3 +608,67 @@ func (a *Authenticate) saveSessionToDataBroker(
 
 	return nil
 }
+
+// FrontchannelLogout signs the user out
+func (a *Authenticate) FrontchannelLogout(w http.ResponseWriter, r *http.Request) error {
+	ctx, span := trace.StartSpan(r.Context(), "authenticate.FrontchannelLogout")
+	defer span.End()
+
+	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
+	w.Header().Set("Cache-Control", "no-cache, no-store")
+	w.Header().Set("Pragma", "no-cache")
+
+	if err := a.processFrontchannelLogoutWithContext(ctx, w, r); err != nil {
+		log.FromRequest(r).Debug().Err(err).Msg("failed processing frontchannel logout")
+
+		w.WriteHeader(http.StatusBadRequest)
+		fmt.Fprintf(w, "%s", "mising required parameters")
+
+		return nil
```

Comment:
We actually wrap the standard HTTP request handler to return an error page / response. You should be able to return the error here directly with any wrapped message.

Reply:
since nobody is interested in the result of the call, I removed the BadRequest response. Showing the errors in the logs should be enough.